java/build.gradle.kts

```diff
@@ -7,7 +10,7 @@ version = "1.0-SNAPSHOT"
 
 java {
     toolchain {
-        languageVersion.set(JavaLanguageVersion.of(21))
```

Comment:
I wonder why using a specific version of 21 is not working. Supposedly this should make gradle download the necessary version. Is using the `current` version the right approach?